src/components/TextInput/TextInput.tsx

```diff
@@ -89,11 +94,13 @@ export const TextInput = ({
     readonly,
     focusOnMount,
     selectable = false,
+    extraAction,
```

Comment:
It is really not obvious what is `extraAction`

Reply:
Do you have any suggestion @SamuelAlev?

Reply:
I adopted the name from the property on the figma Text Input

Reply:
It is only fine, if `clearable` and `copyable` move over there... which is what the designs reflects seems like:
![Screenshot 2023-04-18 at 11 57 39](https://user-images.githubusercontent.com/6846950/232742589-013610fe-1fe5-4176-a11f-b54ca0c9dc03.png)

Reply:
Then just make sure the signature is already the same as the one you plan to use (so probably an array)